docs/user-guide/configuring/configuration-files.md

```diff
@@ -105,7 +105,7 @@ your-project
   └── test.js
 ```
 
```

Comment:
Looks like we are missing the backticks around the “.eslintrc” and “root: true”

Reply:
Fixed it.